infra/modules/storage/metadata.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     source:
       repo: https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf.git
       sourceType: git
-      dir: /infra/modules/storage
+      dir: storage
```

Comment:
Is "storage" correct?

If it's relevant to the repo root, then "/infra/modules/storage" looks more correct.

Reply:
All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.